Osoti12exercises.ipynb

```diff
@@ -0,0 +1,626 @@
+{
```

Comment:
<p>Nice job adding the type checking here to mark these as integers! Did you check what happens if you provide something that is not an integer? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ihmeuw/vivarium_research/pull/1626/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>